src/elements/save-for-later/src/index.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsx jsx */
+import * as React from 'react'
+import { jsx } from 'theme-ui'
+import { Icon } from '@uswitch/trustyle.icon'
+
+interface Props extends React.HTMLAttributes<any> {
+  checked: boolean
+  size?: number
+  className?: string
+}
+
+const SaveForLater: React.FC<Props> = ({
+  checked,
+  onClick,
+  className,
+  size = 40
+}) => {
+  return (
+    <label sx={{ display: 'block' }}>
+      <input
+        sx={{
+          marginLeft: '-9000px',
+          appearance: 'none',
+          position: 'absolute'
+        }}
+        type="checkbox"
+      />
```

Comment:
`onClick` + `checked` should be wired up here instead so it works for non-mouse users

Reply:
oh, thanks!